
Dafny program verifier finished with 0 verified and 0 failed assertions
externErrors.dfy(12,36): Error: inside a module that replaces another, {:extern} attributes may only be used without arguments
